src/NuGet.Core/NuGet.Credentials/NuGet.Credentials.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>NuGet client's authentication models</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
```

Comment:
Update the property in the common targets.

Reply:
Thanks! Changed accordingly.